EIPS/eip-1328.md

```diff
@@ -42,6 +42,14 @@ wc:8a5e5bdc-a0e4-4702-ba63-8f1a5655744f@1?bridge=https%3A%2F%2Fbridge.walletconn
 
 The need for this ERC stems from the discussion to move away from JSON format used in the alpha version of the WalletConnect protocol which makes for very inneficient parsing of the intent of the QR code, making it easier to create better QR code parsers APIs for Wallets to implement. Also by using a URI instead of a JSON inside the QR-Code the Android Intent system can be leveraged.
 
+## Backwards Compatibility
+
+Versioning is required as part of the syntax for this URI specification to allow WalletConnect protocol to evolve and be backwards-compatible whenver a new version instroduced
+
+## Security Considerations
+
+URIs should be shared between user devices or applications and no sensitive data is shared within the URI that could compromise the communication or would allow control of the user's private keys
```

Comment:
```suggestion
URIs should be shared between user devices or applications and no sensitive data is shared within the URI that could compromise the communication or would allow control of the user's private keys.
```